src/pre_commit_vauxoo/pre_commit_vauxoo.py

```diff
@@ -74,6 +80,8 @@ def copy_cfg_files(
                         "Disabling the following pylint checks (PYLINT_DISABLE_CHECKS): %s", pylint_disable_checks
                     )
                     line = line.replace("R0000", ",".join(pylint_disable_checks))
+                if fname == "pyproject.toml" and line.startswith("skip-string-normalization"):
+                    line = "skip-string-normalization=%s" % (skip_string_normalization and "false" or "true")
```

Comment:
For readability, I'd use:

```suggestion
                    line = "skip-string-normalization=%s" % str(skip_string_normalization).lower() 
```

Reply:
This way could allow any string injection

Reply:
Good point.

What do you think about something like a `bool2str` method? Similar to MQT. It could be helpful on future cases

Reply:
My first commit was using [str2bool](https://github.com/Vauxoo/maintainer-quality-tools/blob/9f62ee05bd4b70407fa4c707a051861034ead56f/travis/test_server.py#L117) but `click` is doing the same for us returning a boolean `True` for "true", "True", "1" and so on

But even the boolean needs to be transformed to string again `str(boolean).lower()` looks good in this case

But the method could be called from another side directly bypassing click and injecting any string here

Also `skip_string_normalization and "true" or "false"` is readability too for me

Reply:
In the future we could use the new python style `def copy_cfg_files(..., exclude_autofix: bool)` to be sure it is a boolean

Similar to
 - https://github.com/PyCQA/pylint/blob/ef7a9108e1bc5d1052d41bf7eda0311610c39d73/pylint/checkers/base_checker.py#L112

However, this style could raise compatibility issues